src/components/Forms/_FormElement.js

```diff
@@ -36,13 +49,13 @@ const FormElement = ({ Label, Value, Type, onChange, onSwitch, ImageHandler }) =
           case "Switch":
             return (
               <div className="form-check form-switch">
-                <div>
+                <div className="" onClick={hoverClick}>
```

Comment:
No need to use a `div` here, make it label with `display: block`, will fix it. That's the fix. No need to think so much with `useState()` and stuff.